examples/plugin_example/README.md

```diff
@@ -1 +1,130 @@
-An example application to show how to extend girder_worker to allow for custom celery based distributedjob management that doesn't rely on girder_worker.run. 
+# Girder Worker Example Plugin
```

Comment:
This repeats the RTD docs. I recommend we remove the docs here or else (maybe point to the RTD docs instead) or else one will get out of date.